app/controllers/api/projects.php

```diff
@@ -468,12 +470,13 @@
     ->label('sdk.response.model', Response::MODEL_PROJECT)
     ->param('projectId', '', new UID(), 'Project unique ID.')
     ->param('service', '', new WhiteList(array_keys(array_filter(Config::getParam('services'), function($element) {return $element['optional'];})), true), 'Service name.')
-    ->param('status', null, new Boolean(), 'Service status.')
+    ->param('status', true, new Boolean(), 'Service status.')
```

Comment:
What's the reasoning behind changing this default value? We should be careful when changing default behavior 🤔

Reply:
Great catch. It is left-over from putting services into sub-collection which was later cancelled. Will remove.